forte/data/caster.py

```diff
@@ -78,3 +78,43 @@ def input_pack_type():
     @staticmethod
     def output_pack_type():
         return MultiPack
+
+
+class DataPackBoxer(Caster[MultiPack, DataPack]):
+    """
+    This class creates a DataPack from a MultiPack, this DataPack is the only
+    contain of the original MultiPack, indexed by the :attr:`pack_name`.
+    """
+
+    def cast(self, pack: MultiPack) -> DataPack:
+        """
+        Auto-box the multi-pack into a data-pack by simple getting pack.
```

Comment:
So we don't refer to them as `MultiPack` or `DataPack`, sometimes `Data Pack`. Let's keep this consistent.